R/databricks_connection.R

```diff
@@ -40,6 +40,11 @@ databricks_connection <- function(config, extensions) {
     }
   )
 
+  # Hand in driver's libPaths to worker if user hasn't already set libpaths for notebook-scoped libraries
+  if (is.null(config$spark.r.libpaths)) {
+    config$spark.r.libpaths <- toString(.libPaths())
+  }
```

Comment:
```suggestion
    config$spark.r.libpaths <- config$spark.r.libpaths %||% paste(.libPaths(), collapse = ",")
```

Reply:
We also typically shorten the

```
if (is.null(x))
  x <- expr
```
type of assignment into `x <- x %||% expr` in sparklyr code base using the `%||%` operator from rlang.